src/test/data/JsonUserPrefsStorageTest/UndersizedWindowUserPref.json

```diff
@@ -0,0 +1,11 @@
+{
+  "guiSettings" : {
+    "windowWidth" : 1000.0,
+    "windowHeight" : 400.0,
```

Comment:
```suggestion
    "windowWidth" : 1280.0,
    "windowHeight" : 720.0,
```

Reply:
no, this is for the test to make sure it coerces bad settings to at least the min height/width; nothing wrong with this